lib/node_modules/@stdlib/math/base/complex/divide/lib/index.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+/**
+* Divide two complex numbers.
+*
+* @module @stdlib/math/base/complex/divide
+*
+* @example
+* var cdiv = require( '@stdlib/math/base/complex/divide' );
+*
+* var v = cdiv( -13.0, -1.0, -2.0, 1.0 );
+* // returns [ 5.0, 3.0 ]
+*/
```

Comment:
Yes, there is a decent amount of doc duplication; however, different contexts end up having slightly different conventions and/or emphasis. For example, in the REPL, examples are often formatted differently and may not be exhaustive. In README, we may explore and exhaust all options, but not do so in JSDoc. So, while we recognize the pain and additional burden, we are also leery about constantly having to address the need for customization, rendering automation only marginally useful. For simpler examples, yes, automation could provide benefit, so long as examples have a 1:1 correspondence across contexts, but, in general, the belief is that this will not hold true, even for a large enough minority of cases.